demos/projects/NXP/mimxrt1060/gcc_flags.cmake

```diff
@@ -30,6 +30,8 @@ set(CMAKE_C_FLAGS ${CMAKE_C_FLAGS} " \
 set(CMAKE_EXE_LINKER_FLAGS ${CMAKE_EXE_LINKER_FLAGS} " \
     -fno-exceptions \
     -fno-rtti \
```

Comment:
nano-spec is missing in both compile and linker steps. Can you also please make sure that map file shows the correct library is getting linked (and test...:))

Reply:
yup lib nano is linked

Reply:
![image](https://user-images.githubusercontent.com/36710865/125859303-e21af86f-22e3-4164-9405-396f431b9a43.png)